bin/pulsar-admin

```diff
@@ -24,4 +24,4 @@ export PULSAR_HOME=`cd -P $BINDIR/..;pwd`
 
 #Change to PULSAR_HOME to support relative paths
 cd "$PULSAR_HOME"
-exec $JAVA $OPTS org.apache.pulsar.admin.cli.PulsarAdminTool $PULSAR_CLIENT_CONF "$@"
+exec $JAVA --add-opens java.base/java.lang=ALL-UNNAMED $OPTS org.apache.pulsar.admin.cli.PulsarAdminTool $PULSAR_CLIENT_CONF "$@"
```

Comment:
this must be put in `pulsar-admin-common.sh` as it checks the java version

Reply:
done